From c28dd47eec849835c7690a4d63f91be11aebf887 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Wed, 19 Feb 2014 15:04:51 +0100
Subject: [PATCH 1/3] spl_mmc.c: Display correct message when bitstream fails
 to load

When fpga.bin is missing it displayed this confusing message:

  reading fpga.bin
  spl: error reading image system.dtb, err - -1

Fix this in code by reporting CONFIG_SPL_FPGA_LOAD_ARGS_NAME.
---
 common/spl/spl_mmc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/spl/spl_mmc.c b/common/spl/spl_mmc.c
index a1703f0..35eedcc 100644
--- a/common/spl/spl_mmc.c
+++ b/common/spl/spl_mmc.c
@@ -86,7 +86,7 @@ static int mmc_load_fpga_image_fat(struct mmc *mmc)
 	if (err <= 0) {
 #ifdef CONFIG_SPL_LIBCOMMON_SUPPORT
 		printf("spl: error reading image %s, err - %d\n",
-		       CONFIG_SPL_FAT_LOAD_ARGS_NAME, err);
+		       CONFIG_SPL_FPGA_LOAD_ARGS_NAME, err);
 #endif
 		return -1;
 	}
-- 
1.7.9.5

